src/klish-plugin-infix/xml/infix.xml

```diff
@@ -154,7 +165,16 @@
 <COMMAND name="copy" help="Copy configuration, e.g., copy running-config startup-config">
   <PARAM name="src" ptype="/DATASTORE"    help="Source datastore"/>
   <PARAM name="dst" ptype="/RW_DATASTORE" help="Destination datastore"/>
-  <ACTION sym="copy@infix"/>
+  <ACTION sym="copy@infix" in="tty" out="tty" interrupt="true"/>
+</COMMAND>
+
+<COMMAND name="dir" help="List available configuration files">
+  <ACTION sym="dir@infix"/>
+</COMMAND>
+
+<COMMAND name="erase" help="Erase a configuration file">
+  <PARAM name="file" ptype="/FILES" help="Configuration file"/>
+  <ACTION sym="erase@infix" in="tty" out="tty" interrupt="true"/>
 </COMMAND>
```

Comment:
`erase` feels a litte idiosyncratic. `remove`?

Reply:
Agreed, `remove` it is.  Changing and merging to main, thanks! :tada: